src/components/Layout.tsx

```diff
@@ -1,15 +1,22 @@
 import { Box } from '@mui/material';
 
 interface LayoutProps {
-    children: React.ReactNode;
+  children: React.ReactNode;
 }
 
 const Layout = ({ children }: LayoutProps) => {
-    return (
-        <Box minHeight={'calc(100vh - (2*64px))'} pb={1}>
-            {children}
-        </Box>
-    );
+  return (
+    <Box
+      sx={(theme) => ({
```

Comment:
Was your intention just to contract the search box or to reduce the whole layout to 80% of the original with when the viewport is above 1200px?
Because this would imply that you affect the entire layout with this change, the problematic search box width is not targeted directly but indirectly.

Also, on this branch main content does not honor gaps between the result items.
<img width="1422" alt="Screenshot 2024-02-02 at 12 24 29" src="https://github.com/e1401/musicolor/assets/38010322/62968e2f-e45f-4624-92ff-b84e1be7e54f">

And left alignment of the starting screen feels a bit odd. Should we move that to the centre? What do you suggest?
<img width="1671" alt="Screenshot 2024-02-02 at 12 23 25" src="https://github.com/e1401/musicolor/assets/38010322/6513aeb6-38c1-44fa-a6d2-677ed6eada97">

Reply:
@MatejaC we cannot merge without resolving a conversation